### ISSUE TITLE
Pointer error when running `devtools::test()`

### ISSUE THREAD

**Issue by kenahoo** [oracle/fastr#28]
I have a company-internal package called `AMGUtils` that I wanted to test under FastR.  Here's what happens:

```
> devtools::test()
Loading AMGUtils
Error in match(x, table, nomatch = 0L) :
  cannot coerce type 'externalptr' to vector of type 'character'

> traceback()
6: match(x, table, nomatch = 0L)
5: libs %in% pkglibs
4: .dynLibs(libs[!(libs %in% pkglibs)]) at unload.r#119
3: unload_dll(pkg) at load.r#135
2: load_all(pkg, quiet = TRUE)
1: devtools::test()
```

**Comment by steve-s** [comment]
Hi Ken, 

this should be fixed in b5066aaec2f2cfddf6ecf2036b78f2e6bdc86f8b. I've tried running the tests of `assertthat` via `devtools::tests()` and they seem to work fine. Again, to try this out now, you can use the community provided [Dockerfile](https://github.com/nuest/fastr-docker), or wait for the next release.

**Comment by kenahoo** [comment]
Thanks!
